.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+### General (all pull requests)
+
+- [ ] I have read the [contribution guide](https://osu.ppy.sh/wiki/en/osu%21_wiki_Contribution_Guide)
+
+This pull request:
+
+- [ ] conforms to the [formatting](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting) and [writing](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting) style criteria;
+- [ ] is related to an existing issue (specify which one).
+
+### Translations only
+
+This pull request:
+
+- [ ] does not introduce any original content that does not exist in the original article(s);
+- [ ] will be reviewed by native speakers.
```

Comment:
Would probably specify when that is the case (guessing formatting/markup changes only)?

Reply:
nah, this is the main reason for the PR: we want translations to almost always [follow the original article](https://github.com/ppy/osu-wiki/blob/master/wiki/Article_styling_criteria/Formatting/en.md#content-parity), with exception to language-specific information. thinking of how to phrase that better

Reply:
Ah sorry brainfart, meant "when that is *not* the case".